### PR TITLE
Enable dependabot.

### PR DESCRIPTION
**Pull request by qwandor**
This should help us keep on top of dependency version updates.

### CODE REVIEW

**Comment by jalil-salame** [comment]
In this case merging is blocked by a clippy lint which looks like a manual impl of an async mutex... LGTM but that needs to be investigated.

**Comment by jalil-salame** [comment]
Seemed straight forward enough to fix: #57

If it gets merged you can rebase this branch on top of it to make the CI happy c:

**Comment by jalil-salame** [comment]
Rebased on top of master, now the workflows should pass

**Review by jalil-salame — APPROVED** [review]
LGTM

File: .github/workflows/commit-message-check.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
 
     - name: DCO Check
-      uses: tim-actions/dco@2fd0504dc0d27b33f542867c300c60840c6dcb20
+      uses: tim-actions/dco@v1.1.0
```
[the remark this answers is not in the document]
Comment (qwandor) [reply .github/workflows/commit-message-check.yaml:27]: This is cleanup, it's easier to see what's going on depending on a tag rather than a commit hash.

**Review by Tim-Zhang — APPROVED** [review]

**Pull request merged**